dynamics-nav/Server-Driven-Paging-in-OData-Web-Services.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Server-Driven Paging in OData Web Services"
+description: Server-driven paging ensures that the quantity of data that is returned by an OData URI does not overwhelm nav_server or client program that you use to capture data.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.